convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -922,6 +936,38 @@ def test_is_disable_and_enable_repos_doesnt_thas_same_repo(self):
         pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
         self.assertEqual(logger.CustomLogger.warning.called, 0)
 
+    @unit_tests.mock(system_info, "name", "Oracle Linux Server release 7.9")
+    @unit_tests.mock(system_info, "arch", "x86_64")
+    @unit_tests.mock(system_info, "version", "7")
+    @unit_tests.mock(logger.CustomLogger, "info", LogMocked())
+    @unit_tests.mock(utils, "get_file_content", lambda x: "UPDATEDEFAULT=yes\nDEFAULTKERNEL=kernel-uek\n")
+    @unit_tests.mock(utils, "store_content_to_file", StoreContentToFileMocked())
+    def test_fix_default_kernel_converting_oracle(self):
+        pkghandler.fix_default_kernel()
+        self.assertTrue(logger.CustomLogger.info.called)
+        self.assertTrue("/etc/sysconfig/kernel", utils.store_content_to_file.filename)
+        self.assertTrue("DEFAULTKERNEL=kernel" in utils.store_content_to_file.content)
+        self.assertFalse("DEFAULTKERNEL=kernel-uek" in utils.store_content_to_file.content)
+
+        system_info.name = "Oracle Linux Server release 8.1"
+        system_info.version = "8"
+        pkghandler.fix_default_kernel()
+        self.assertTrue("DEFAULTKERNEL=kernel" in utils.store_content_to_file.content)
```

Comment:
```suggestion
        self.assertTrue("DEFAULTKERNEL=kernelcore" in utils.store_content_to_file.content)
```

Reply:
`kernelcore` is not a valid flavor for default kernels.